examples/tfengine/generated/folder_foundation/live/prod/team1/folder/terragrunt.hcl

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include {
+  path = find_in_parent_folders()
+}
+
+dependency "parent_folder" {
+  config_path = "../../folder"
```

Comment:
I wonder is this config_path always holds? Can users put the parent folder in a wrong directory and this is not valid?

Reply:
Yeah that's a good question.. in that case I guess it just wouldn't work and the user may need to set their own custom terraform_addon type thing. We can support that if the need arises, but this way kind of forces our layout standardization which can be useful.